nilearn/input_data/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,219 @@
+"""
+Transformer for computing ROI signals of multiple 4D images
+"""
+
+import collections
+import itertools
+import warnings
+
+from nilearn._utils.compat import Memory, Parallel, delayed
+
+from .. import _utils
+from .. import image
+from .. import masking
+from .._utils import CacheMixin
+from .._utils.class_inspect import get_params
+from .._utils.compat import _basestring, izip
+from .._utils.niimg_conversions import _iter_check_niimg
+from .nifti_labels_masker import NiftiLabelsMasker
+from nilearn.image import get_data
+
+
+class MultiNiftiLabelsMasker(NiftiLabelsMasker, CacheMixin):
+    """Class for masking of Niimg-like objects.
+
+    MultiNiftiLabelsMasker is useful when data from non-overlapping volumes and from
+    different subjects should be extracted (contrarily to NiftiMapsMasker). Use case:
```

Comment:
```suggestion
    different subjects should be extracted (contrarily to NiftiLabelsMasker). Use case:
```
Is that what you meant?

Reply:
"contrary to"